tv_apps/browser/locales-obj/ar.json

```diff
@@ -0,0 +1 @@
_SETTINGS","$v":"إعدادات المتصفح"},{"$i":"WB_LT_CLEAR","$v":"محو"},{"$i":"WB_LT_CLEAR_ALL_HISTORY","$v":"مسح كل السجلات؟"},{"$i":"WB_LT_CLEAR_BROWSING_HISTORY","$v":"مسح سجل التصفح"},{"$i":"WB_LT_CLEAR_HISTORY","$v":"مسح السجل"},{"$i":"WB_LT_CLEAR_STORED_DATA","$v":"مسح الكوكيز والبيانات المسجلة"},{"$i":"WB_LT_CONFIRM_DELETE_COOKIES","$v":"هل ترغب في حذف ملفات تعريف الارتباط؟"},{"$i":"WB_LT_CONNECTION_IS_UNTRUSTED","$v":"هذا التوصيل غير موثوق به"},{"$i":"WB_LT_CURSOR_MODE","$v":"وضع المؤشر"},{"$i":"WB_LT_HOMEPAGE","$v":"الصفحة الرئيسية"},{"$i":"WB_LT_LEARN_MORE","$v":"معرفة المزيد"},{"$i":"WB_LT_PAGE_CANNOT_BE_DISPLAYED","$v":"لا يمكن عرض هذه الصفحة"},{"$i":"WB_LT_PAN_MODE","$v":"وضع السحب"},{"$i":"WB_LT_PRESS_COLOR_KEY","$v":"اضغط على مفتاح الاختصار الملون على الجهاز للتحويل بين وضع المؤشر ووضع السحب."},{"$i":"WB_LT_PRIVATE_BROWSING_1","$v":"لن يتذكر Firefox أي سجل لهذه النافذة."},{"$i":"WB_LT_PRIVATE_BROWSING_2","$v":"في نافذة التصفح الخاص، لن يبقي Firefox أي سجل متصفح، سجل بحث، سجل تحميل، سجل على شكل ويب، كوكيز أو ملفات إنترنت مؤقتة. ومع ذلك، سوف يتم الحفاظ على الملفات التي تقوم بتحميلها والتفضيلات.<BR><BR> لإيقاف التصفح الخاص، يمكنك إغلاق هذه النافذة."},{"$i":"WB_LT_PRIVATE_BROWSING_3","$v":"بينما لن يوجد في هذا الكمبيوتر سجل تصفح، يمكن لمزود خدمة الإنترنت أو صاحب العمل تتبع الصفحات التي زرتها."},{"$i":"WB_LT_REMOVE","$v":"حذف"},{"$i":"WB_LT_REMOVE_FROM_BOOKMARKS","$v":"مسح صفحة الويب من التفضيلات؟"},{"$i":"WB_LT_REMOVE_FROM_TOP_SITES","$v":"مسح من المواقع العليا"},{"$i":"WB_LT_REMOVE_HISTORY","$v":"حذف السجل؟"},{"$i":"WB_LT_RENAME_BOOKMARK","$v":"إعادة تسمية التفضيلات؟"},{"$i":"WB_LT_RESTORE_TO_DEFAULT","$v":"الإعادة للوضع الافتراضي"},{"$i":"WB_LT_SETTINGS_PRIVATE_BROWSING","$v":"تصفح خاص"},{"$i":"WB_LT_SETTINGS_SEARCH_ENGINE","$v":"محرك البحث"},{"$i":"WB_LT_SET_AS_HOMEPAGE","$v":"ضبط كصفحة رئيسية"},{"$i":"WB_LT_SET_HOMEPAGE","$v":["تم ضبط ",{"t":"idOrVar","v":"value0"}," الآن كصفحتك الرئيسية"]},{"$i":"WB_LT_SWITCH_CURSOR_MODE","$v":"لقد قمت بالتحويل إلى وضع المؤشر"},{"$i":"WB_LT_SWITCH_PAN_MODE","$v":"لقد قمت بالتحويل إلى وضع السحب"},{"$i":"WB_LT_TIPS_BOOKMARK_THIS_PAGE","$v":"إضافة هذه الصفحة للتفضيلات"},{"$i":"WB_LT_TIPS_CHANGE_SCREEN","$v":"تغيير الشاشة"},{"$i":"WB_LT_TIPS_HISTORY_BACK","$v":"الرجوع إلى السجل الزمني"},{"$i":"WB_LT_TIPS_HISTORY_FORWARD","$v":"تقديم السجل الزمني"},{"$i":"WB_LT_TIPS_HOMEPAGE","$v":"عرض الصفحة الرئيسية"},{"$i":"WB_LT_TIPS_MENU","$v":"القائمة"},{"$i":"WB_LT_TIPS_NEW_PRIVATE_WINDOW","$v":"نافذة خاصة جديدة"},{"$i":"WB_LT_TIPS_OPEN_A_NEW_TAB","$v":"فتح علامة تبويب جديدة"},{"$i":"WB_LT_TIPS_RELOAD","$v":"إعادة التحميل"},{"$i":"WB_LT_TIPS_SEARCH_BAR","$v":"قم بالبحث على الويب"},{"$i":"WB_LT_TIPS_SHOW_YOUR_BOOKMARK","$v":"عرض التفضيلات"},{"$i":"WB_LT_TIPS_STOP_LOADING","$v":"إيقاف فتح هذه الصفحة"},{"$i":"WB_LT_TIPS_S_SEARCH","$v":[{"t":"idOrVar","v":"value0"}," بحث"]},{"$i":"WB_LT_TIPS_TABS","$v":"عرض علامات التبويب"},{"$i":"WB_LT_TIPS_ZOOM_IN","$v":"تكبير"},{"$i":"WB_LT_W_BOOKMARK_DELETE","$v":"حذف علامة الموقع"}]
```

Comment:
We may need to remove files under `tv_apps/browser/locales-obj/` and use locales/en-US.properties instead.